katdal/sensordata.py

```diff
@@ -73,7 +73,7 @@ def __bool__(self):
         return len(self.timestamp) > 0
 
 
-class SensorData(object):
+class SensorGetter(object):
     """Raw (uninterpolated) sensor data placeholder.
 
     This is basically a placeholder for uninterpolated sensor data resembling
```

Comment:
This sentence now sounds out of place. Maybe remove it and change the next sentence to:

```
This is an abstract lazy interface that provides a :class:`SensorData` object on request but does not store values itself.
```

Reply:
Good point. I've taken that and welded it in to the rest of the docstring. See what you think now.